Load all files in a data directory: "_all.yml" and other files if present.

PaulPichaureau opened the pull request:
Sometimes it could be interesting to have a "_all.yml" files for many little pages and some file for big pages.

I propose to load all the files, plain and simple. :-)

Commit:
Fixes #64 - last tag registered was invoked for all tag invocations

thanethomson committed:
bumping version to v0.21.3

thanethomson committed:
adding CHANGELOG entry for v0.21.3

PaulPichaureau committed:
Load all files in a data directory: "_all.yml" and other files if present.

PaulPichaureau committed:
Load all files in a data directory: "_all.yml" and other files if present.

PaulPichaureau committed:
Content field from _all.yml files are now loaded.

PaulPichaureau committed:
Content field from _all.yml files are now loaded.

PaulPichaureau committed:
Check if field is present in case of ManyToMany relation

PaulPichaureau committed:
Merge branch 'debug2'

Commit:
Updates code to allow for newest version of Markdown library (> 3.0)

Commit:
Fixes requirements to latest versions of libraries

Commit:
Removes broken import

Commit:
Updates changelog for v0.22.0

Commit:
Updates version to v0.22.0

Commit:
Fixes missing paramiko requirement

Commit:
Bumps version to v0.22.1

Commit:
Updates changelog for v0.22.1

thanethomson committed:
Updates CHANGELOG for v0.22.2

thanethomson committed:
Bumps version to v0.22.2

thanethomson committed:
Adds Python 3.7 to Travis CI build

thanethomson committed:
Tries Tornado Travis CI approach to be able to build Python 3.7

thanethomson committed:
Removes CONTRIBUTORS file in favour of just going to https://github.com/thanethomson/statik/graphs/contributors (less overhead)

thanethomson committed:
Removes CONTRIBUTORS file in favour of just going to https://github.com/thanethomson/statik/graphs/contributors (less overhead)

thanethomson committed:
Updates package tags to include Python 3.7 for next release

jayvdb commented:
I've created https://github.com/thanethomson/statik/pull/89 and https://github.com/thanethomson/statik/pull/90 with the two enhancements in this PR.  As far as I can see, that is the only two enhancements in here, that are not already merged.

CLiu13 committed:
database.py: Preserve default list value order

This ensures that the default list value order
will be the order in the resulting output.

Fixes https://github.com/thanethomson/statik/issues/76

kx-chen committed:
Allow access to variables defined in config in views

Context defined in the config can now be accessed in views

Closes https://github.com/thanethomson/statik/issues/54

kx-chen committed:
database.py: Allow for multiple foreign keys in same document

Allow for multiple foreign keys in the same document by telling
SQLAlchemy which foreign key to look for.

Closes https://github.com/thanethomson/statik/issues/27

PaulPichaureau committed:
clean

PaulPichaureau committed:
Revert "Change yaml load to full_load. See https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation"

This reverts commit 4a96b8e4dc6b367a87b97f5a7a82e9d428b637d5.

PaulPichaureau committed:
Change yaml load to full_load. See https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

PaulPichaureau committed:
Change yaml load to full_load. See https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

PaulPichaureau committed:
Ignore pipfile

PaulPichaureau committed:
Disable markdown extension

PaulPichaureau committed:
Logging des données non trouvées lors du build

PaulPichaureau committed:
clean

PaulPichaureau committed:
Add warnings

PaulPichaureau committed:
Changement dans la gestion de _all.yml

Désormais les fichiers *_.yml sont chargés comme les _all.yml. Il peut donc y en avoir plusieurs.

PaulPichaureau committed:
La version précise qu'on est en "PP"

PaulPichaureau committed:
Change in name of *.yml

PaulPichaureau committed:
Merge pull request #2 from PaulPichaureau/master-pp

Master pp
